server/websocket/websocket_connector.go

```diff
@@ -138,10 +158,18 @@ func (ws *WebSocket) sendLoop() {
 }
 
 func (ws *WebSocket) checkAccess(raw []byte) bool {
-	protocol.Debug("websocket: raw message: %v", string(raw))
+	logger.WithFields(log.Fields{
```

Comment:
- might check for log level debug, before , to avoid processing/converting to string large messages
- code here always returns true anyway, and raw message might have been already logged elsewhere - so logging here might be removed ?